src/Presenters/BulmaPresenter.php

```diff
@@ -0,0 +1,428 @@
+            if ($this->element->multiple) {
+                $this->element->addClass('checkbox');
+            } else {
+                $this->element->addClass('radio');
+            }
+        } elseif ($this->element instanceof Select) {
+            $this->element->addClass('select');
+        } elseif ($this->element instanceof TextArea) {
+            $this->element->addClass('textarea');
+        }
+    }
+
+    /**
+     * @return string
+     */
+    public function renderField()
+    {
+        $this->addElementClasses();
+
+        if ($this->error) {
+            $this->element->addClass('is-danger');
+        }
+
+        $rendered = $this->element->render();
+
+        if(is_array($rendered)) {
+            $isInline = $this->isInline() || $this->element->inline;
+
+            return $isInline ? $this->renderInlineList($rendered) : $this->renderList($rendered);
+        }
+
+        if ($this->isSelect()) {
+            $errorClass = $this->error ? ' is-danger' : '';
+            $rendered = '<div class="select is-fullwidth' . $errorClass . '">' . $rendered . '</div>';
+        }
+        
+        return $rendered;
+    }
+
+    /**
+     * @param array $elements
+     * @return string
+     */
+    protected function renderInlineList($elements)
+    {
+        $class = $this->element->multiple ? $this->style['label-checkbox'] : $this->style['label-radio'];
+        $class .= $this->error ? ' has-text-danger' : '';
+        $list = '';
+
+        foreach($elements as $element) {
+            $list .= '<label class="' . $class . '">' . $element . '</label>';
+        }
+
+        return $list;
+    }
+
+    /**
+     * @param array $elements
+     * @return string
+     */
+    protected function renderList($elements)
+    {
+        $list = '';
+        $class = $this->element->multiple ? $this->style['label-checkbox'] : $this->style['label-radio'];
+        $class .= $this->error ? ' has-text-danger' : '';
+
+        foreach($elements as $element) {
+            $list .= '<div><label class="' . $class . '">' . $element . '</label></div>';
+        }
+
+        return $list;
+    }
+
+    /**
+     * @return string
+     */
+    public function renderError()
+    {
+        return $this->error ? '<strong>' . $this->error . '</strong>' : '';
+    }
+
+    /**
+     * @return string
+     */
+    public function renderAll()
+    {
+        $this->element->addGroupClass($this->getGroupClass());
+
+        $result = '';
+
+        $result .= '<div' . $this->renderAttributes($this->element->groupAttributes) . '>';
+        $result .= $this->renderElementGroup();
+        $result .= '</div>';
+
+        return $result;
+    }
+
+    /**
+     * @return string
+     */
+    protected function renderElementGroup()
+    {
+        if($this->isCheckbox()) return $this->renderCheckbox();
+
+        $result = $this->renderLabel();
+
+        $result .= $this->renderPrefix() . $this->renderField() . $this->renderSuffix();
```

Comment:
Tukaj bi samo prosil če preveriš, ali se prefix in suffix pravilno dodata na select element? Mogoče zna biti težava, ker renderField() zavije select v div.